apps/react/src/features/auth/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable max-lines-per-function */
+import { memo, useEffect, FC } from 'react';
+import { Field, Form, FormikProvider, useFormik } from 'formik';
+import { TextField } from 'formik-mui';
+import { Alert, Box } from '@mui/material';
+import { LoadingButton } from '@mui/lab';
+import { useAppDispatch, useAppSelector } from '@js-camp/react/store/store';
+import { AuthActions } from '@js-camp/react/store/auth/dispatchers';
+import { Login } from '@js-camp/core/models/login';
+import { AppValidationError } from '@js-camp/core/models/app-error';
+import { PasswordField } from '@js-camp/react/components';
+
+import { initialValues, loginFormSchema, LoginFormValue } from './form-setting';
+
+const LoginFormComponent: FC = () => {
+  const dispatch = useAppDispatch();
+  const { error, isLoading } = useAppSelector(state => state.auth);
+
+  useEffect(() => {
+    dispatch(AuthActions.resetAuthErrorAndLoading());
+    return () => {
+      dispatch(AuthActions.resetAuthErrorAndLoading());
+    };
+  }, []);
+
+  const handleLoginButtonClick = (value: LoginFormValue): void => {
+    dispatch(AuthActions.loginUser(value));
+  };
+
```

Comment:
If am not mistaken, writing return type is not necessary here
```suggestion

  const handleLoginButtonClick = (value: LoginFormValue) => {
    dispatch(AuthActions.loginUser(value));
  };

```